docker-compose/all/docker-compose.yml

```diff
@@ -0,0 +1,174 @@
+version: '3'
```

Comment:
Can we add a comment at the top of the file indicating which networks are included in `all`?